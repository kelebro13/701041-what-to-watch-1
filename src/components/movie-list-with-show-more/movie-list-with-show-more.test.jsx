import renderer from "react-test-renderer";
import MovieListWithShowMore from "./movie-list-with-show-more";

it(`renders properly`, () => {

  const films = new Array(20);

  const tree = renderer
    .create(<MovieListWithShowMore films={films} countItem={20}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
