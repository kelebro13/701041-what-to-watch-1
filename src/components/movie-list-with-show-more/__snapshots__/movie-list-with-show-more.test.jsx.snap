// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="catalog__movies-list"
/>
`;
